docs/core-concepts/run.mdx

```diff
@@ -7,48 +7,115 @@ toc_min_heading_level: 2
 toc_max_heading_level: 4
 ---
 
-All UDFs have a secure HTTP endpoint callable via a [web request](/core-concepts/run/#http-requests) or [`fused.run`](/core-concepts/run/#fusedrun) that is authorized with a [shared](/core-concepts/run/#shared-token) or [private](/core-concepts/run/#private-token) token. The endpoint's response may be [cached](/core-concepts/run/#caching-responses) and have its type [specified](/core-concepts/run/#response-data-types).
+{/* All UDFs have a secure HTTP endpoint callable via a [web request](/core-concepts/run/#http-requests) or [`fused.run`](/core-concepts/run/#fusedrun) that is authorized with a [shared](/core-concepts/run/#shared-token) or [private](/core-concepts/run/#private-token) token. The endpoint's response may be [cached](/core-concepts/run/#caching-responses) and have its type [specified](/core-concepts/run/#response-data-types). */}
+
+Fused UDF functions really shine once you start calling them from anywhere. You can do so in 2 main ways:
+1.  `fused.run()` in Python. All you need is [`fused` Python](/python-sdk/) package is installed
```

Comment:
```suggestion
1.  `fused.run()` in Python. All you need is having the [`fused` Python](/python-sdk/) package installed
```

(there was a double `is`, not entirely sure my rewording is the best english)

Reply:
Good catch, updated